### PR TITLE
Hemant k/maq changes

### PR DESCRIPTION
**Pull request by maqhemantk**
•	Muted the Video player by default
•	Paused background slider when popup window is open for Power BI slider on the home

### CODE REVIEW

File: js/theme.js [hunk 1]
```diff
@@ -44,6 +44,12 @@ $(document).ready(function () {
         live: true
     })
     wow.init();
+    $(".product_view").on('shown.bs.modal', function (event) {
```
Comment (maqsoftwaremain) [inline js/theme.js]: increase specificity

File: js/theme.js [hunk 2]
```diff
@@ -44,6 +44,12 @@ $(document).ready(function () {
         live: true
     })
     wow.init();
+    $(".product_view").on('shown.bs.modal', function (event) {
+        $(".power-bi-carousel").trigger('stop.owl.autoplay');
+    });
+    $(".product_view").on('hidden.bs.modal', function (event) {
```
Comment (maqsoftwaremain) [inline js/theme.js]: same as above

File: js/theme.js (hunk 1 above)
Comment (maqhemantk) [reply js/theme.js]: Updated

File: js/theme.js (hunk 2 above)
Comment (maqhemantk) [reply js/theme.js]: Updated

**Pull request merged**